### ISSUE TITLE
Consider providing global styles

### ISSUE THREAD

**Issue by earshinov** [madoar/angular-archwizard#199]
Hello @madoar ,

Do you think it would be useful if angular-archwizard provided "global styles"? That is, instead of compiling styles into the library, the library would provide *.css / *.scss / *.less files to be included by the application.

# Motivation

This way library users could customize the appearance of the wizard by setting LESS / SCSS variables instead of overriding CSS styles with `!important`, which is a bit messy. An example from our project:

```scss
aw-wizard-navigation-bar.large-empty-symbols ul.steps-indicator li.optional,
aw-wizard-navigation-bar.large-empty-symbols ul.steps-indicator li.done,
aw-wizard-navigation-bar.large-empty-symbols ul.steps-indicator li.navigable {
  .step-indicator {
    color: $app-theme-text-green !important;
    border: 2px solid $app-theme-green500 !important;
  }
  a:hover .step-indicator {
    color: $app-theme-green700 !important;
    border: 2px solid $app-theme-green700 !important;
  }
  a:hover .label {
    color: #4d4d4d !important;
  }
}
```

I believe many users of angular-archwizard customize its colors to match the application UI (although I do not have any statistics obviously).

This approach is used by large frameworks like Bootstrap or Angular Material, and by individual components like [ng-select](https://github.com/ng-select/ng-select/#step-3-include-a-theme) as well.

A downside is that this change will be BACKWARDS INCOMPATIBLE. Users will have to import angular-archwizard's styles explicitly, otherwise wizards' appearance will break after the upgrade.

# Implementation

@angular/cli does not (and will not) support global styles: https://github.com/angular/angular-cli/issues/10869

However, it should be fairly easy to implement a custom solution using:
- [less-bundle](https://www.npmjs.com/package/less-bundle) to bundle multiple *.less files to a single one
- [less](https://www.npmjs.com/package/less) to compile LESS to CSS (for users who would like to include css)
- [less2sass](https://www.npmjs.com/package/less2sass) for converting LESS to SCSS (for people who prefer SCSS)

less2sass is a weak point here, because it will not provide a perfect conversion. Maybe angular-archwizard could switch to SCSS? I believe that most UI frameworks (Angular Material and Bootstrap 4 to name a few) and components either have been using SCSS from the start or have migrated to it.

# Useful Links

- https://medium.com/@Dor3nz/compiling-css-in-new-angular-6-libraries-26f80274d8e5

**Comment by madoar** [comment]
### Global CSS:
If you mean by `global css` an external css file, like angular material [provides](https://material.angular.io/guide/getting-started#step-4-include-a-theme) provides, then I agree. Providing the styles via an external file would be very useful.

In the last weeks I briefly thought about changing the way the styles are added to the components (i.e. from a per component approach to a global one). My main issues were first the amount of time this takes, and second the fact that this represents a very large breaking change in my opinion, which deterred me previously.

### Change from LESS to SCSS
Again I have no problem with this :) I previously used LESS because it is the first stylesheet dialect except CSS I've learned. I'm not really attached to it. As long as we're not using straight CSS I'm basically alright with it.

**Comment by madoar** [comment]
This has been implemented in #200
